packages/web/src/scss/components/Modal/index.html

```diff
@@ -1,37 +1,270 @@
 {{#> layout/plain }}
 
+  <section class="docs-Section">
+
+    <h2 class="docs-Heading">Modal</h2>
+
+    <button
+      type="button"
+      class="Button Button--primary Button--medium"
+      data-toggle="modal"
+      data-target="#example_1"
+      aria-controls="example_1"
+      aria-expanded="false"
+    >
+      Open Modal
+    </button>
+
+    <!-- Modal: start -->
+    <dialog id="example_1" class="Modal Modal--composed" aria-labelledby="example_1_title">
+      <div class="Modal__dialog">
+
+        <!-- Modal Header: start -->
+        <div class="ModalHeader">
```

Comment:
Can we use `<header class="ModalHeader">` element instead of `div`?

Reply:
I was considering that, it's not a topic as straightforward as it might seem… But, after some reserach and consulting with @crishpeen, I made the inner structure of the Modal more semantic.